tests/python/relax/test_relax_tensor_ops.py

```diff
@@ -605,5 +605,64 @@ def expected(x: R.Tensor((2, 64, 8, 9), "float32")) -> R.Tensor(None, "float32",
     tvm.ir.assert_structural_equal(bb.get()["main"], expected)
 
 
+def test_sigmoid():
+    @R.function
+    def expected(x: R.Tensor((2, 3), "float32")) -> R.Tensor(None, "float32", ndim=2):
+        gv: R.Tensor((2, 3), "float32") = R.sigmoid(x)
+        return gv
+
+    x = relax.Var("x", [2, 3], relax.DynTensorType(ndim=2, dtype="float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.sigmoid(x))
+        bb.emit_func_output(gv)
+
+    expected = expected.with_attr("global_symbol", "main")
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_less():
+    @R.function
+    def expected(
+        x: R.Tensor((2, 3), "float32"), y: R.Tensor((2, 1), "float32")
+    ) -> R.Tensor(None, "bool", ndim=2):
+        gv: R.Tensor((2, 3), "bool") = R.less(x, y)
+        return gv
+
+    x = relax.Var("x", [2, 3], relax.DynTensorType(ndim=2, dtype="float32"))
+    y = relax.Var("y", [2, 1], relax.DynTensorType(ndim=2, dtype="float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x, y]):
+        gv = bb.emit(relax.op.less(x, y))
+        bb.emit_func_output(gv)
+
+    expected = expected.with_attr("global_symbol", "main")
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_cross_entropy_and_softmax_cross_entropy():
+
+    def _test_single_op(R_func, relax_op_nn_func):
+        @R.function
+        def expected(
+            predictions: R.Tensor((2, 3), "float32"), targets: R.Tensor((2, 3), "float32")
+        ) -> R.Tensor(None, "float32", ndim=0):
+            gv: R.Tensor((), "float32") = R_func(predictions, targets)
+            return gv
+
+        predictions = relax.Var("predictions", [2, 3], relax.DynTensorType(ndim=2, dtype="float32"))
+        targets = relax.Var("targets", [2, 3], relax.DynTensorType(ndim=2, dtype="float32"))
+        bb = relax.BlockBuilder()
+        with bb.function("main", [predictions, targets]):
+            gv = bb.emit(relax_op_nn_func(predictions, targets))
+            bb.emit_func_output(gv)
+
+        expected = expected.with_attr("global_symbol", "main")
+        tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+    _test_single_op(R.cross_entropy, relax.op.nn.cross_entropy)
+    _test_single_op(R.softmax_cross_entropy, relax.op.nn.softmax_cross_entropy)
```

Comment:
Would be better to split them separately, since cross entropy and softmax cross entropy are not exactly the same thing.